react/features/base/avatar/components/web/StatelessAvatar.js

```diff
@@ -40,6 +43,8 @@ export default class StatelessAvatar extends AbstractStatelessAvatar<Props> {
      */
     render() {
         const { initials, url } = this.props;
+        const avatars = new Avatars(sprites, AVATAR_CONFIG);
+        const svg = avatars.create(initials);
```

Comment:
this is better moved in the `if` where it is used, tough the var itself is redundant if it is only used once